ReadMe.md

```diff
@@ -55,69 +55,69 @@ For CLI usage please see [Manual.md](Manual.md).
+  upload_concurrency: 1          # UPLOAD_CONCURRENCY, max 255, by default, the value is round(sqrt(AVAILABLE_CPU_CORES / 2))
 
-  # RESTORE_SCHEMA_ON_CLUSTER, execute all schema related SQL queries with `ON CLUSTER` clause as Distributed DDL, 
-  # look to `system.clusters` table for proper cluster name, also system.macros could be used, 
-  # don't applicable when `use_embedded_backup_restore: true`
+  # RESTORE_SCHEMA_ON_CLUSTER, execute all schema related SQL queries with `ON CLUSTER` clause as Distributed DDL. 
+  # Check `system.clusters` table for the correct cluster name, also `system.macros` can be used.
+  # This isn't applicable when `use_embedded_backup_restore: true`
   restore_schema_on_cluster: ""   
   upload_by_part: true           # UPLOAD_BY_PART
   download_by_part: true         # DOWNLOAD_BY_PART
-  use_resumable_state: true      # USE_RESUMABLE_STATE, allow resume upload and download according to <backup_name>.resumable file
+  use_resumable_state: true      # USE_RESUMABLE_STATE, allow resume upload and download according to the <backup_name>.resumable file
 
-  # RESTORE_DATABASE_MAPPING, restore rules from backup databases to target databases, which is useful on change destination database all atomic tables will create with new uuid.
-  # for environment use following format "src_db1:target_db1,src_db2:target_db2" 
+  # RESTORE_DATABASE_MAPPING, restore rules from backup databases to target databases, which is useful when changing destination database, all atomic tables will be created with new UUIDs.
+  # Use the following format "src_db1:target_db1,src_db2:target_db2" 
   restore_database_mapping: {}   
-  retries_on_failure: 3          # RETRIES_ON_FAILURE, retry if failure during upload or download
-  retries_pause: 30s             # RETRIES_PAUSE, time duration pause after each download or upload fail 
+  retries_on_failure: 3          # RETRIES_ON_FAILURE, how many times to retry after a failure during upload or download
+  retries_pause: 30s             # RETRIES_PAUSE, duration time to pause after each download or upload failure 
 clickhouse:
   username: default                # CLICKHOUSE_USERNAME
   password: ""                     # CLICKHOUSE_PASSWORD
   host: localhost                  # CLICKHOUSE_HOST
   port: 9000                       # CLICKHOUSE_PORT, don't use 8123, clickhouse-backup doesn't support HTTP protocol
-  # CLICKHOUSE_DISK_MAPPING, use it if your system.disks on restored servers not the same with system.disks on server where backup was created
-  # for environment use following format "disk_name1:disk_path1,disk_name2:disk_path2" 
+  # CLICKHOUSE_DISK_MAPPING, use this mapping when your `system.disks` are different between the source and destination clusters during backup and restore process
+  # Use the following format "disk_name1:disk_path1,disk_name2:disk_path2" 
```

Comment:
no, `for environment` here means for environment variable CLICKHOUSE_DISK_MAPPING

for config file you shall continue to use map in YAML